feat(issues): Update saved search documentation and issue page screenshots (#5768)
getsentry/sentry-docs: src/docs/product/issues/index.mdx

When you click on an issue on the main **Issues** page, the **Issue Details** page for that issue is displayed. Learn more in [Issue Details](/product/issues/issue-details/).

> Sentry can also saved your issue queries on this page, which can be accessed by clicking the "Saved Searches" button in the header. Learn more in [Saved Searches](/product/sentry-basics/search/saved-searches/)

Reviewer: ```suggestion
You can also save your issue queries and access them later by clicking the the "Saved Searches" button in the header. Learn more in [Saved Searches](/product/sentry-basics/search/saved-searches/).
```
